src/immutable_data.rs

```diff
@@ -22,18 +22,20 @@ use xor_name::XorName;
 use sodiumoxide::crypto;
 
 #[derive(Hash, Clone, Eq, PartialEq, Ord, PartialOrd, RustcEncodable, RustcDecodable, Debug)]
-/// ImmutableDataType.
+/// The type of an individual copy of immutable data.
```

Comment:
Done.
(Added it to the documentation of the enum variants and of `ImmutableData`.)